spec/takeaway_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'takeaway'
+require 'menu'
+
+describe Takeaway do
+
+  let(:menu) { Menu.new }
+
+  describe "display_menu" do
+    it "displays the menu" do
+      expect(subject.display_menu).to eq(menu.show_menu)
+    end
+  end
+
+  describe "select_dish" do
+
+    # it "checks if the dish is on the menu" do
+    #   expect(subject.select_dish("Hello Meow")).to be_on_menu
+    # end
+
+    it "adds a dish to the order if it's on the menu" do
+      subject.select_dish("Hello Meow")
+      expect(subject.order).to include("Hello Meow")
+    end
+
```

Comment:
Good SRP-ing here. Removed tests that no longer need to be present when new features added.